lightning/src/ln/functional_test_utils.rs

```diff
@@ -742,14 +742,19 @@ macro_rules! unwrap_send_err {
 }
 
 /// Check whether N channel monitor(s) have been added.
+pub fn check_added_monitors(node: &Node, count: usize) {
+	let mut added_monitors = node.chain_monitor.added_monitors.lock().unwrap();
+	assert_eq!(added_monitors.len(), count);
+	added_monitors.clear();
+}
+
+/// Check whether N channel monitor(s) have been added.
+///
+/// Don't use this, use the identically-named function instead.
```

Comment:
Would `#[deprecated = "Don't use this, use the identically-named function instead"]` be too noisy in the warnings?